src/components/BackButton/StyledBackButton.tsx

```diff
@@ -0,0 +1,20 @@
+import { Button } from '..';
```

Comment:
Nit: an alternate way to organize these components in the file structure would be to put all buttons in the same folder called `src/components/Buttons`. You could then have an index.ts file to export the normal button, BackButton, etc. But we can always organize later as more components are added.